src/pages/creations/index.tsx

```diff
@@ -49,15 +66,26 @@ export default function Creations() {
             </HeroDivider>
 
             <TitleContainer>
-              <ProjectTitle title='Partner projects' />
+              <ProjectTitle>
```

Comment:
Current version:  
![image](https://github.com/user-attachments/assets/5e485f66-8faf-4b9f-bb32-a770c094bf80)

After discussing with Mati, we thought it might be better to display the small carousel as soon as you enter. So, we suggest removing the "We work with the best" section, resulting in something like this:  
![image](https://github.com/user-attachments/assets/763cf02c-24c9-4ea5-bd33-111518271a67)

Reply:
https://github.com/defi-wonderland/web/pull/114/commits/bd489c9aa3a93b955c137ca49e16d9083e8521ad
https://github.com/defi-wonderland/web/pull/114/commits/826c27c17b2f3040f3a683ec3a34966a9360e81e